cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/async/progress/OperationProgress.java

```diff
@@ -84,6 +86,21 @@ public synchronized String toString() {
     return sb.toString();
   }
 
+  public synchronized Map<String, Map<String, String>> getJsonStructure() {
```

Comment:
Why do we return a `Map<String, Map<String, String>>` and not `Map<String, Object>`? It might be useful to check the format of the other functions that are performing the similar operation.